### PR TITLE
UI updates

### LINKED ISSUE #152: Tabs have different shapes
Small issue, Home tab has different shape from the rest of the tabs
image
image

### PR DESCRIPTION
**Pull request by noelmathewisaac**
- Fixed rounding of boxes
- Fixed the status tab colours
- Renames Tasks tab to Home tab

### CODE REVIEW

**Review by CSmortal — APPROVED** [review]
LGTM

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -51,39 +51,31 @@
...
                                       </padding>
                                       <StackPane fx:id="completedTaskListPanelPlaceholder" VBox.vgrow="ALWAYS">
-                                          <VBox.margin>
-                                              <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
-                                          </VBox.margin>
                                       </StackPane>
                                   </VBox>
                               </Tab>
-                              <Tab fx:id="tabUncompleted" closable="false" text="Uncompleted" disable="true">
+                              <Tab fx:id="tabUncompleted" closable="false" disable="true" text="Uncompleted">
```
Comment (hengyongming) [inline src/main/resources/view/MainWindow.fxml:73]: What's the difference haha?

**Review by hengyongming — APPROVED** [review]
LGTM

File: src/main/resources/view/MainWindow.fxml (hunk 1 above)
Comment (noelmathewisaac) [reply src/main/resources/view/MainWindow.fxml:73]: I think scene builder automatically changed haha

**Pull request merged**